requirements-dev.txt

```diff
@@ -0,0 +1,6 @@
+-r requirements.txt
+mock
+pep8==1.6.2
+pytest
+pytest-cov
+pytest-pep8
```

Comment:
There's actually a `docs/requirements.txt` as well.  I guess I'll try and consolidate the two then.